app.rb

```diff
@@ -11,91 +11,116 @@ def initialize
     @rentals = []
   end
 
-  def list_all_books(_books)
+  def list_all_books
     @books.each do |book|
       puts "Title: #{book.title}, Author: #{book.author}"
       puts ' '
     end
   end
 
-  def list_all_people(_people)
+  def list_all_people
     @people.each do |person|
       print "[#{person.class}] name: #{person.name}, ID: #{person.id}, #{person.age}"
       puts ' '
     end
   end
 
-  def create_stedent
+  def age
     print 'Age:'
-    age = gets.chomp
-    print 'name:'
-    name = gets.chomp
+    gets.chomp
+  end
+
+  def name
+    print 'Name:'
+    gets.chomp
+  end
+
+  def permission
     print 'Has parent permission? [Y/N]:'
     permit = gets.chomp
     case permit
     when 'Y', 'y'
-      permission = true
+      true
     when 'N', 'n'
-      permission = false
+      false
     end
+  end
+
+  def create_student
     student = Student.new(age, name, parent_permission: permission)
     @people.push(student)
     puts 'Person successfully created'
   end
```

Comment:
- [ ] Great work refactoring the code and implementing all suggestions made by the former reviewer. After creating a `student`, I noticed that the output was wrong options when I listed people. In your refactor, you forgot to add the input for `classroom` when creating a student. This is critical because in your `Student` class, the first parameter is the `classroom` data. What this means, is that the wrong parameters would be passed for different data when creating a new `Student` instance. Kindly create an input for `classroom` and and it as a first parameter when you are creating a new `Student` instance. Below is the output when I list people using the console 👍 💻 


![image](https://user-images.githubusercontent.com/77106962/218516468-41bf6381-5c7a-4f0c-a79c-b023b1554017.png)